### ISSUE TITLE
Add get_profile feature to pull contact details

### ISSUE THREAD

**Issue by Strykar** [hoehermann/purple-signald#28]
```
@thefinn93 │ Strykar: there should be contact lookup methods? not entirely sure though, new profiles aren't fully implemented yet, sadly
@thefinn93 │ oh actually get_profile should do what you need
```
So this should enable changing the phone numbers to contact names?

**Comment by hoehermann** [comment]
I marked this with "help wanted" as I am not interested in implementing this myself. I have so few contacts using Signal, manually naming them does not bother me.

On a side note: The signal protocol is currently being [reimplemented](https://github.com/signalapp/libsignal-client). Once done, this will likely obsolete libpurple-signald as well as the other approach I tried with [purple-signal](https://github.com/hoehermann/purple-signal). For this reason, I am very hesitant to add new features unless doing so looks fun to me.
